### DISCUSSION TITLE
Can access github with brower normally, but when installing packages, an error will be reported that it cannot be accessed

### DISCUSSION THREAD

**Discussion by HawkKingdom** [LykosAI/StabilityMatrix#703] [Q&A]
dont know why...
Just like the title said: Can access github with brower normally, but when installing packages, an error will be reported that it cannot be accessed
Help Please!
![Dingtalk_20240620234805](https://github.com/LykosAI/StabilityMatrix/assets/35811349/91f4be54-eca5-44b8-bef6-5e0762c862ab)
![Dingtalk_20240620234829](https://github.com/LykosAI/StabilityMatrix/assets/35811349/0713618a-4065-49c9-aeb2-b69a9b55d736)

**Comment by Taozki** [comment]
I wonder if you fixed this issue, I have the same problem
